packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2942,3 +3048,123 @@ class _Editable extends MultiChildRenderObjectWidget {
       ..setPromptRectRange(promptRectRange);
   }
 }
+
+class _ScribbleFocusable extends StatefulWidget {
```

Comment:
Maybe in the long term we can make this public.  I think some users have built rich text editors that don't use EditableText, and they probably want Scribble support ideally.

CC @Renzo-Olivares